Source/Particles/Collision/BinaryCollision/BinaryCollision.H

```diff
@@ -263,9 +247,12 @@ public:
             const auto soa_1 = ptile_1.getParticleTileData();
             index_type* indices_1 = bins_1.permutationPtr();
             index_type const* cell_offsets_1 = bins_1.offsetsPtr();
```

Comment:
Just because we add more pointers now:
```suggestion
            index_type* AMREX_RESTRICT indices_1 = bins_1.permutationPtr();
            index_type const* AMREX_RESTRICT cell_offsets_1 = bins_1.offsetsPtr();
```